tests/test_kinesis/test_kinesis.py

```diff
@@ -479,6 +481,65 @@ def test_get_records_from_empty_stream_at_timestamp():
     response["MillisBehindLatest"].should.equal(0)
 
 
+@mock_kinesis
+def test_valid_increase_stream_retention_period():
+    conn = boto3.client("kinesis", region_name="us-west-2")
+    stream_name = "my_stream"
+    conn.create_stream(StreamName=stream_name, ShardCount=1)
+
+    conn.increase_stream_retention_period(
+        StreamName=stream_name, RetentionPeriodHours=40
+    )
+
+    response = conn.describe_stream(StreamName=stream_name)
+    response["StreamDescription"]["RetentionPeriodHours"].should.equal(40)
+
+
+@mock_kinesis
+def test_invalid_increase_stream_retention_period():
+    conn = boto3.client("kinesis", region_name="us-west-2")
+    stream_name = "my_stream"
+    conn.create_stream(StreamName=stream_name, ShardCount=1)
+
+    conn.increase_stream_retention_period(
+        StreamName=stream_name, RetentionPeriodHours=30
+    )
+    conn.increase_stream_retention_period.when.called_with(
```

Comment:
Negative tests should use pytest.raises, to validate that the error is raised in the correct format:
```
with pytest.raises(ClientError) as ex:
        conn.increase_stream_retention_period(..)
ex.value.response["Error"]["Code"].should.equal("...")
ex.value.response["Error"]["Code"].should.equal("...")
```